src/libs/Navigation/AppNavigator/ModalStackNavigators/useModalScreenOptions.ts

```diff
@@ -0,0 +1,22 @@
+import type {StackNavigationOptions} from '@react-navigation/stack';
+import {CardStyleInterpolators} from '@react-navigation/stack';
+import {useMemo} from 'react';
+import useThemeStyles from '@hooks/useThemeStyles';
+import type {ThemeStyles} from '@src/styles';
+
+function useModalScreenOptions(getScreenOptions?: (styles: ThemeStyles) => StackNavigationOptions) {
```

Comment:
Let's also use this hook in `createModalStackNavigator`

Reply:
fixed https://github.com/Expensify/App/pull/39084/commits/d5765d4a6c6a0cc4818b1ad3a49cf549089833cf 👍